src/lib/change-handlers.ts

```diff
@@ -278,7 +350,10 @@ export const handleUnknownChange = (): SnippetData => {
   }
 }
 
-export const changeOperationAlias = (operation: string): string => {
+export const changeOperationAlias = (
+  operation: string,
+  indentationType?: 'indent' | 'unindent'
```

Comment:
perhaps at this point it would make more sense just to pass the entire dataTracked here. Although I don't insist and it you prefer to disregard, please resolve this comment.

Reply:
Done